utility_network/perform-valve-isolation-trace/src/main/java/com/esri/samples/perform_valve_isolation_trace/PerformValveIsolationTraceController.java

```diff
@@ -303,6 +328,144 @@ private void handleTraceClick() {
     }
   }
 
+  /**
+   *  Uses the clicked map point to identify any utility elements in the utility network at the clicked location. The
+   *  clicked utility element is added either to the utility trace parameter's filter barrier list. A graphic is created
```

Comment:
```suggestion
   *  clicked utility element is added to the utility trace parameter's filter barrier list. A graphic is created
```
Possible correction, or there may be an "either" to add here?